### PR TITLE
feat(fxa-settings): Change fxa-settings default locale from en-US to en

### PR DESCRIPTION
**Pull request by vpomerleau**
Linked to [PR-14429](https://github.com/mozilla/fxa/pull/14429)

Because:

- We are standardizing the default locale for all fxa packages to en.

This commit:

- Change the fluent extraction source from $SETTINGS_DIR/public/locales/en-US/*.ftl to $SETTINGS_DIR/public/locales/en/*.ftl

Closes #FXA-6003

### CODE REVIEW

**Review by bcolsson — APPROVED** [review]
LGTM, combined with the other workflow should not affect the l10n extract output.

**Comment by bcolsson** [comment]
Will hold off merging until the other gets merged.

**Comment by LZoog** [comment]
Other PR was merged, going ahead and merging this now since Bryan r+'d.

**Pull request merged**
